<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ricardo Villalba</author>
 <class>Gui::TLogWindow</class>
 <widget class="QWidget" name="Gui::TLogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Log Window</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QPlainTextEdit" name="edit"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Save log to file</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="toolTip">
        <string>Copy log to clipboard</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="findLabel">
       <property name="text">
        <string>Fi&amp;nd:</string>
       </property>
       <property name="buddy">
        <cstring>findEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="foundLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findNextButton">
       <property name="toolTip">
        <string>Find next (F3)</string>
       </property>
       <property name="text">
        <string>Find next</string>
       </property>
       <property name="shortcut">
        <string>F3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findPreviousButton">
       <property name="toolTip">
        <string>Find previous (Shift+F3)</string>
       </property>
       <property name="text">
        <string>Find previous</string>
       </property>
       <property name="shortcut">
        <string>Shift+F3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>findEdit</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>findPreviousButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
